scripts/install_cvmfs_eessi.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Gather information about the OS
+. /etc/os-release
+echo "Attempting to install EESSI for operating system ($NAME : $VERSION)"
+error_msg="Don't know how to handle operating system ($NAME : $VERSION)"
+# Check whether we have a RHEL-like or Debian-like system
+if [[ $ID_LIKE == *"rhel"* ]] || [[ $ID_LIKE == *"fedora"* ]] || [[ $ID_LIKE == *"centos"* ]]
```

Comment:
I ended up with `[[ "${ID_LIKE}" =~ "fedora" ]]`, which looks cleaner in my eyes, but yeah. Note that RHEL will always (RHEL7 and up at least) have ID_LIKE="fedora" so ironically, the only distributions that have ID_LIKE="rhel" are non-RHEL and non-Fedora clones/copies...

Reply:
I used all three because they appear in AmazonLinux2, everything else just had a single entry. Since they were in AL2, I assumed they may appear elsewhere...but I have no idea TBH.

Reply:
I updated things to that style, and kept the other comparisons around as they do no harm

Reply:
I kinda get the feeling that AL2 threw the kitchen sink at distro "compatibility" here. https://www.freedesktop.org/software/systemd/man/os-release.html is at least a bit helpful, bit real-world vs freedesktop is always a losing battle for the latter. :(